src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,5 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+export const UserInfo = ({ user }) => (
```

Comment:
There should be a null or undefined check for 'user' prop before using it. If user is null or undefined, it could potentially break the component rendering.